packages/swap/prisma/schema.prisma

```diff
@@ -165,3 +165,13 @@ model ThirdPartySwap {
 
   @@schema("public")
 }
+
+model Pool {
+  id               Int   @id @default(autoincrement())
+  baseAsset        Asset
+  pairAsset        Asset
+  feeHundredthPips Int
```

Comment:
should we call this liquidityFee or is that implied ?

Reply:
i called it `feeHundredthPips` because that is how it is called in the protocol, but will change it to `liquidityFeeHundredthPips`